avtozip/store/api/resources.py

```diff
@@ -0,0 +1,16 @@
+from tastypie.resources import ModelResource
+from store.models import Product
+from django.http import Http404
+
+
+class ProductResource(ModelResource):
+    class Meta:
+        queryset = Product.objects.all()
+        allowed_methods = ['post', 'put', 'get', 'patch', 'delete']
+        filtering = {
+            'name': ('exact', 'icontains',),
```

Comment:
Remove contains by now. I will create new task to substitute exact with contains using custom filters